src/user/views/adsManager/views/advanced/components/form/EditCampaign.tsx

```diff
@@ -22,7 +22,7 @@ interface Props {
 }
 
 interface Params {
-  campaignId: string
+  campaignId: string;
```

Comment:
Could you raise an issue to set up prettier (with enforcement and a pre-commit hook) on this codebase like we do on ads-admin please?